test/Board/Queen.test.ts

```diff
@@ -0,0 +1,252 @@
+import { Board } from "../../src/Board/Board";
+import { Pawn } from "../../src/Board/Pawn";
+import { PieceColor } from "../../src/Board/PieceColorEnum";
+import { PieceType } from "../../src/Board/PieceTypeEnum";
+import { Queen } from "../../src/Board/Queen";
+import { Square } from "../../src/Board/Square";
+import { Game } from "../../src/Game/Game";
+import { Move } from "../../src/Game/Move";
+
+
+describe("Moves for queen", () => {
+    //TEST 1-1
+    test("Queen can move on free square", () => {
+        //given
+        const queen = new Queen(PieceColor.White, PieceType.Queen, new Square(0, 0));
+        const board = new Board();
+        board.addPiece(queen);
+        const game = new Game(undefined, undefined, undefined, undefined, undefined, board);
```

Comment:
Czy nie powinno być tak?
```suggestion
        const game = new Game(undefined, PieceColor.White, undefined, undefined, undefined, board);
```

Tak mi ostatnio @Chlip zaproponował w moim PRze

Reply:
no i oczywiście, to się tyczy wszystkich przypadków poniżej również

Reply:
już poprawione